### PR TITLE
Connect Asset Tree to SDK

### PR DESCRIPTION
**Pull request by follan**
#150

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-161.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-161.gearboxjs.preview.cogniteapp.com

File: src/components/AssetTree/stories/AssetTree.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    query: sdk.AssetListDescendantsParams
+  ) => {
+    return {
+      items: ASSET_LIST_CHILD.sort((a, b) => (a.id === assetId ? -1 : 1)),
+    };
+  };
+};
+const zeroChild = ASSET_ZERO_DEPTH_ARRAY.findIndex(asset => asset.depth === 0);
+
+storiesOf('AssetTree', module).add('Full description', () => <AssetTree />, {
```
Comment (mrtraser) [inline src/components/AssetTree/stories/AssetTree.stories.tsx]: Guess we need `setupMocks()` here as well

File: src/components/AssetTree/stories/AssetTree.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    info: {
+      header: false,
+      source: false,
+      styles: {
+        infoBody: { display: 'none' },
+      },
+    },
+  })
+  .add(
+    'basic',
```
Comment (mrtraser) [inline src/components/AssetTree/stories/AssetTree.stories.tsx]: Better looks with capitalize letter "B" -> Basic

Comment (follan) [reply src/components/AssetTree/stories/AssetTree.stories.tsx]: Fixed

File: src/components/AssetTree/stories/AssetTree.stories.tsx (hunk 1 above)
Comment (follan) [reply src/components/AssetTree/stories/AssetTree.stories.tsx]: Fixed

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-161.gearboxjs.preview.cogniteapp.com

**Pull request merged**